README.md

```diff
@@ -52,6 +52,16 @@ or use the [dojo cli](https://github.com/dojo/cli-create-app) to create a comple
 dojo-widgets are based on a virtual DOM implementation named [Maquette](http://maquettejs.org/) as well as some foundational classes
 provided in [dojo-compose](https://github.com/dojo/compose).
 
+The smallest `dojo-widgets` example looks like this:
+
+```ts
+const projector = createProjector();
+projector.children = d('h1', 'Hello, Dojo!');
```

Comment:
It wasn't valid, have updated to `projector.children = [ d('h1', [ 'Hello, Dojo!' ]); ]`